tests/tests_pytorch/callbacks/progress/test_rich_progress_bar.py

```diff
@@ -416,3 +417,17 @@ def test_rich_progress_bar_padding():
     train_description = progress_bar._get_train_description(current_epoch=0)
     assert "Epoch 0/0" in train_description
     assert len(progress_bar.validation_description) == len(train_description)
+
+
+def test_rich_progress_bar_can_be_pickled():
+    bar = RichProgressBar()
+    trainer = Trainer(fast_dev_run=True, callbacks=[bar], max_steps=1)
```

Comment:
Mind adding the usual boilerplate of `logger=None, enable_checkpointing=False, enable_model_summary=False`...

Reply:
I copied the test function from the tqdm tests. I will keep the checkpointing=True since this test is about pickling.